File: Java/백준/Bronze/28113. 정보섬의 대중교통/정보섬의 대중교통.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int n = in.nextInt(); int a = in.nextInt(); int b = in.nextInt(); int ck=1;
        if(n>b){
            ck=0;
        }
        if(b==a && ck==1){
            System.out.println("Anything");
        }
        else if(b<a && ck==1){
            System.out.println("Subway");
        }
        else{
            System.out.println("Bus");
        }
    }
}

File: C++11/백준/Bronze/28113. 정보섬의 대중교통/정보섬의 대중교통.cc
#include <stdio.h>

int main(){
    int a,b,c;
    scanf("%d %d %d",&a,&b,&c);
    if(a<=c){
        if(b==c){
            printf("Anything");
        }
        else if(b>c){
            printf("Subway");
        }
        else{
            printf("Bus");
        }
    }
    else{
        printf("Bus");
    }
}